src/app/app.component.spec.ts

```diff
@@ -155,4 +158,68 @@ describe('AppComponent', () => {
        flush();
   }));
 
+  // spinner is shown when user is authorized
+
+  it('should hide the spinner, after workspace is (intially) loaded', fakeAsync(() => {
+    resetCalls(mockNg4SpinnerService);
+    when(mockIndexService.reload()).thenReturn(Promise.resolve());
+    when(mockIndexService.refresh()).thenReturn(Promise.resolve([]));
+    when(mockValidationMarkerService.getAllMarkerSummaries(anything())).thenReturn(Promise.resolve([]));
+
+    // when
+    messagingService.publish('workspace.reload.request', null);
+    tick();
+    const [closure] = capture(mockPersistenceService.listFiles).last();
```

Comment:
`closure` seems a bit generic. Would something like `onFilesReceived` be appropriate?

… also I was confused, at first, because I thought `last()` would return the last parameter of `listFiles`' invocation, but it seems to be the capture of the last recorded invocation, right?